proto/zitadel/auth.proto

```diff
@@ -869,9 +874,8 @@ service AuthService {
         };
 
         option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
-            tags: "User Authorizations/Grants"
-            summary: "List My Authorizations/Grants";
-            description: "Returns a list of the authorizations/user grants the authenticated user has. User grants consist of an organization, a project and 1-n roles."
+            tags: "User Authorizations/Grants";
+            deprecated: true;
```

Comment:
remove deprecation?

Reply:
Oh yes, wanted to do that

Reply:
we remove it on https://github.com/zitadel/zitadel/tree/user-profile-requests